Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectDetails.swift

```diff
@@ -38,8 +38,7 @@ struct PPOProjectDetails: View {
             .lineLimit(Constants.titleLineLimit)
         }
         if let symbol = pledge.symbol, let amount = pledge.amount {
-          // TODO: Localize
-          Text("\(symbol)\(amount) pledged")
+          Text(Strings.Pledge_amount_pledged(pledge_amount: "\(symbol)\(amount)"))
```

Comment:
Nit: Should this be using a `currency(` or `formattedCurrency(` helper (in Format.swift) to construct the "\(symbol)\(amount)" string? Ex for backers that don't have USD as their default currency, we usually show "US$" as the currency symbol, and I doubt we're handling that correctly right now. Maybe the view model could expose a `formattedAmount` string that we construct there?

Reply:
[Ticketed](https://kickstarter.atlassian.net/browse/MBL-1878) this for now. The model uses GraphAPI.MoneyFragment internally, so this would require making a bunch of changes just to support this formatting type. I'd recommend instead adding a formatter for MoneyFragment, which would need testing.